scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+set -xxe
+
+SELF="$0"
+ROOTFS_PATH="$(pwd)/bin/targets/x86/64/openwrt-x86-64-generic-rootfs.tar.gz"
+NETWORK_ENABLE="${NETWORK_ENABLE:-0}"
+NETWORK_PREFIX="${NETWORK_PREFIX:-192.168.1}"
+IMAGE_NAME="openwrt-rootfs:$NETWORK_PREFIX"
+NETWORK_NAME="none"
+
+die() {
+	echo "$1"
+	exit 1
+}
+
+usage() {
+	cat >&2 <<EOF
+Usage: $SELF [-h|--help]
+       $SELF
+         [--rootfs <rootfs>]
+         [-n|--network]
+		 [-p|--prebuild]
+
+<rootfs> allows to specifiy a different path for the rootfs.
+<network> enables network access based on <NETWORK_PREFIX>
+<prebuild> uses the official docker images openwrtorg/rootfs:latest
+	-> changes to <NETWORK_PREFIX> are ignored
+EOF
+}
+
+parse_args() {
+	while [ "$#" -gt 0 ]; do
+		case "$1" in
+			--rootfs) ROOTFS_PATH="$2"; shift 2 ;;
+			--network|-n) NETWORK_ENABLE=1; shift ;;
+			--prebuild|-p) PREBUILD=1; shift ;;
+			--help|-h)
+				usage
+				exit 0
+				;;
+			*)
+				DOCKER_EXTRA="$DOCKER_EXTRA $1"
+				shift
+				;;
+		esac
+	done
+}
+
+parse_args "$@"
+
+[ -f "$ROOTFS_PATH" ] || die "Couldn't find rootfs at $ROOTFS_PATH"
+
+if [ -z "$PREBUILD" ]; then
+	cat <<EOT > "$(dirname $ROOTFS_PATH)/Dockerfile"
```

Comment:
Sorry, I've missed this previously, but I think, that it would be nicer to use something like mktemp for this temporary Dockerfile, otherwise you might accidentally overwrite something important :)

Reply:
Good point! Added